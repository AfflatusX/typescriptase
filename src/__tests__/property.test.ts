import { property1, property2, property3 } from "./examples";

test(
  "property1",
  async (): Promise<void> => {
    expect(property1.print({
      name: "property1",
      path: "src/__tests__/property.test.ts",
    }))
      .toMatchSnapshot();
  },
);

test(
  "property2",
  async (): Promise<void> => {
    expect(property2.print({
      name: "property2",
      path: "src/__tests__/property.test.ts",
    }))
      .toMatchSnapshot();
  },
);

test(
  "property3",
  async (): Promise<void> => {
    expect(property3.print({
      name: "property3",
      path: "src/__tests__/property.test.ts",
    }))
      .toMatchSnapshot();
  },
);
