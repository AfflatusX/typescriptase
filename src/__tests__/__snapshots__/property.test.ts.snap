// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`property1 1`] = `
"public property1: string = \\"MYVAR\\";
"
`;

exports[`property2 1`] = `
"protected readonly property2: number;
"
`;

exports[`property3 1`] = `
"private static property3: string | null;
"
`;
